### DISCUSSION TITLE
Some images don't display on the mobile app

### DISCUSSION THREAD

**Discussion by MarvNC** [community/community#48760] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Some of the images in [this repository's](https://github.com/MarvNC/yomichan-dictionaries) readme don't seem to display on Github Mobile for Android. The only one that displays is this: `![](!images/kanjiCssImage.png)` which makes me suspect it has to do with the characters in the filenames and url encoding or something.

**Comment by mattruggio** [comment, marked as answer]
Thanks @MarvNC for reporting this!  Since this repository is public, I took the liberty of [forking](https://github.com/mattruggio/yomichan-dictionaries) this repository and see if it was the image filenames that was causing encoding/decoding issues (diff [here](https://github.com/MarvNC/yomichan-dictionaries/compare/master...mattruggio:yomichan-dictionaries:fix-image-links)).  After removing all URI escaped characters from the name and updating the README to reflect these new name changes I was able to get all nine images to show on mobile (when viewing my forked repositories branch README.md).

I believe you have stumbled upon a mobile encoding/decoding issue with images/names and I will file an internal bug on this.  In the meantime, if you wish for these to display on mobile, I would suggest removing all the escaped/encoded characters from the image filenames.  I do not have the context for this exact repository so I am not sure I would feel comfortable submitting a PR with these fixes for you but feel free to take what you need from my public [fork](https://github.com/mattruggio/yomichan-dictionaries)/[diff](https://github.com/MarvNC/yomichan-dictionaries/compare/master...mattruggio:yomichan-dictionaries:fix-image-links).

**Reply by MarvNC** [reply]
Thanks for testing and following up! I'll probably go change the filenames to be more semantic and add alt text as I should have done earlier, hoping the issue gets fixed 🙏
